quint/src/graphics.ts

```diff
@@ -123,6 +121,99 @@ export function prettyQuintEx(ex: QuintEx): Doc {
   }
 }
 
+export function prettyQuintDef(
+  def: QuintDef,
+  includeBody: boolean = true,
+  type: TypeScheme | undefined = undefined
+): Doc {
+  const typeAnnotation = isAnnotatedDef(def)
+    ? [text(': '), prettyQuintType(def.typeAnnotation)]
+    : type
+    ? [text(': '), prettyTypeScheme(type)]
+    : []
+  switch (def.kind) {
+    case 'def': {
+      const header = group([
+        richtext(chalk.magenta, qualifierToString(def.qualifier)),
+        text(' '),
+        richtext(chalk.blue, def.name),
+        ...typeAnnotation,
+      ])
+      return includeBody ? group([header, text(' = '), prettyQuintEx(def.expr)]) : header
+    }
+    default:
+      // TODO, not used for now
+      return text(definitionToString(def))
```

Comment:
would it be better to return something like `<error>`, since this is an unexpected case?